root/static/scripts/edit/confirmNavigationFallback.js

```diff
@@ -35,9 +35,13 @@ MB.confirmNavigationFallback = function () {
       const node = event.srcElement || event.target;
       const tag = node.tagName.toLowerCase();
       const type = (node.type || '').toLowerCase();
-      const prevent = !((tag == 'input' && (type == 'text' || type == 'password')) || tag == 'textarea');
+      const prevent = !(
+        (tag == 'input' && (type == 'text' || type == 'password')) ||
+        tag == 'textarea');
```

Comment:
I don't know if this violates any rule but I'd keep parentheses consistent with `brace-style`
```suggestion
        tag == 'textarea'
      );
```